src/TodoListItem.jsx

```diff
@@ -0,0 +1,6 @@
+function TodoListItem({ todo }) {
+    return <li>{todo.id}{todo.space}{todo.title} </li>
```

Comment:
Is `todo.space` the best way to achieve creating a space inside and HTML element? Remember that you can render strings inside of HTML element along side expressions.